tensorboard/defs/defs.bzl

```diff
@@ -150,28 +150,45 @@ def tf_ng_module(assets = [], **kwargs):
         **kwargs
     )
 
-def tf_res_digest_suffixer(name, resources, template, out):
+def tf_resource_digest_suffixer(name, resources, template, out):
     """Query parameter suffixer for a resource in a template.
 
     In order to facilitate resource caching, the macro suffixes resource declaration in
     a template file by replacing it with one with query parameter,
-    "?_file_md5=[len_8_truncated_md5]".
+    "?_file_hash=[len_8_truncated_md5]".
 
     For example, if a template is "index.html" that contains content like below,
 
         <script src="index.js"></script>
 
     For resources of ["index.js"], it will be replaced with:
 
-        <script src="index.js?_file_md5=486c31fc"></script>
+        <script src="index.js?_file_hash=486c31fc"></script>
+
+    Args:
+        name: Name of the rule.
+        resources: dict of replacement keyword to resource File.
```

Comment:
Writing `File` here seems a bit confusing, since [`File`] is a Bazel
concept but these values are just strings (this is a macro, not a rule).
Maybe “file” or “label”?

[`File`]: https://docs.bazel.build/versions/master/skylark/lib/File.html